rllib/examples/offline_rl/train_w_bc_finetune_w_ppo.py

```diff
@@ -0,0 +1,284 @@
+    like PPO.
+    - shows how to plug this new PPO-capable RLModule (including its checkpointed state
+    from the BC run) into your algorithm's config.
+    - confirms that even after 1-2 training iterations with PPO, no catastrophic
+    forgetting occurs (due to the additional value function branch and the switched
+    optimizer).
+    - uses Tune and RLlib to continue training the model until a higher return of B
+    is reached.
+
+
+How to run this script
+----------------------
+`python [script file name].py --enable-new-api-stack`
+
+For debugging, use the following additional command line options
+`--no-tune --num-env-runners=0`
+which should allow you to set breakpoints anywhere in the RLlib code and
+have the execution stop there for inspection and debugging.
+
+For logging to your WandB account, use:
+`--wandb-key=[your WandB API key] --wandb-project=[some project name]
+--wandb-run-name=[optional: WandB run name (within the defined project)]`
+
+
+Results to expect
+-----------------
+In the console output, you can first see BC's performance until return A is reached:
++----------------------------+------------+----------------+--------+
+| Trial name                 | status     | loc            |   iter |
+|                            |            |                |        |
+|----------------------------+------------+----------------+--------+
+| BC_CartPole-v1_95ba0_00000 | TERMINATED | 127.0.0.1:1515 |     51 |
++----------------------------+------------+----------------+--------+
++------------------+------------------------+------------------------+
+|   total time (s) |    episode_return_mean |   num_env_steps_traine |
+|                  |                        |             d_lifetime |
+|------------------+------------------------|------------------------|
+|          11.4828 |                  250.5 |                  42394 |
++------------------+------------------------+------------------------+
+
+The script should confirm that no catastrophic forgetting has taken place:
+
+PPO return after initialization: 292.3
+PPO return after 2x training: 276.85
+
+Then, after PPO training, you should see something like this (higher return):
++-----------------------------+------------+----------------+--------+
+| Trial name                  | status     | loc            |   iter |
+|                             |            |                |        |
+|-----------------------------+------------+----------------+--------+
+| PPO_CartPole-v1_e07ac_00000 | TERMINATED | 127.0.0.1:6032 |     37 |
++-----------------------------+------------+----------------+--------+
+
++------------------+------------------------+------------------------+
+|   total time (s) |    episode_return_mean |  num_episodes_lifetime |
+|                  |                        |                        |
++------------------+------------------------+------------------------+
+|          32.7647 |                 450.76 |                    406 |
```

Comment:
Dom we have by case a number how long it takes to train PPO from zero to 450?

Reply:
It's probably not much slower, if at all. But I think this is not the main point here (we know PPO is super fast learning CartPole). The main goal here is to show that:
a) you can use simple custom Models w/o having to sub-class algo-specific RLModule classes!! <- this is huge and thanks to the new RLModule API concept.
b) it doesn't tank (catastrophic forgetting) after transfer from BC to PPO :)